yt_dlp/extractor/eplus.py

```diff
@@ -29,14 +34,97 @@ class EplusIbIE(InfoExtractor):
             'No video formats found!',
             'Requested format is not available',
         ],
+    }, {
+        'url': 'https://live.eplus.jp/sample',
+        'info_dict': {
+            'id': 'stream1ng20210719-test-005',
+            'title': 'Online streaming test for DRM',
+            'live_status': 'was_live',
+            'release_date': '20210719',
+            'release_timestamp': 1626703200,
+            'description': None,
+        },
+        'params': {
+            'skip_download': True,
+            'ignore_no_formats_error': True,
+        },
+        'expected_warnings': [
+            'Could not find the playlist URL. This event may not be accessible',
+            'No video formats found!',
+            'Requested format is not available',
+            'This video is DRM protected',
+        ],
+    }, {
+        'url': 'https://live.eplus.jp/2053935',
+        'info_dict': {
+            'id': '331320-0001-001',
+            'title': '丘みどり2020配信LIVE Vol.2 ～秋麗～ 【Streaming+(配信チケット)】',
+            'live_status': 'was_live',
+            'release_date': '20200920',
+            'release_timestamp': 1600596000,
+        },
+        'params': {
+            'skip_download': True,
+            'ignore_no_formats_error': True,
+        },
+        'expected_warnings': [
+            'Could not find the playlist URL. This event may not be accessible',
+            'No video formats found!',
+            'Requested format is not available',
+        ],
     }]
 
+    _USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36 Edg/119.0.0.0'
+
+    def _login(self, username, password, urlh):
+        if not self._get_cookies('https://live.eplus.jp/').get('ci_session'):
+            raise ExtractorError('Unable to get ci_session cookie')
+
+        cltft_token = urlh.headers.get('X-CLTFT-Token')
+        if not cltft_token:
+            raise ExtractorError('Unable to get X-CLTFT-Token', expected=False)
```

Comment:
`expected=False` is default
```suggestion
            raise ExtractorError('Unable to get X-CLTFT-Token')
```